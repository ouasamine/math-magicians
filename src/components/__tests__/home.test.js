import { render } from '@testing-library/react';
import React from 'react';
import Home from '../Home';

it('Home renders correctly', () => {
  const Component = render(<Home />);
  expect(Component).toMatchSnapshot();
});
