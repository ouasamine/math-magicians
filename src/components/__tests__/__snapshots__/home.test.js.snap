// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h2
        class="home-title"
      >
        Welcome to our page
      </h2>
      <p
        class="home-text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sed vestibulum erat. Nulla sagittis massa at egestas facilisis. Quisque tincidunt interdum vulputate. Sed orci orci, viverra vel tincidunt vel, consectetur eget tortor. Integer pellentesque eros vel ex maximus, quis aliquet nisl tristique. Nulla lobortis sem a placerat maximus. Duis suscipit purus at nunc bibendum, et finibus quam tempor. Donec sed magna suscipit, bibendum nisi faucibus, dictum erat. Aenean varius eros vitae arcu tempus lobortis. Ut et nisl eget purus ornare tincidunt. Aliquam erat volutpat. Suspendisse vitae finibus odio. Suspendisse aliquam risus massa, vel vulputate mauris tempus a. In ullamcorper arcu eu suscipit facilisis. Suspendisse at elit et arcu finibus pulvinar. Fusce at ullamcorper libero. Nunc eget diam rutrum, lobortis sem vel, cursus lacus. Aliquam at elit nec est ultrices pharetra sit amet et dolor. Pellentesque mattis sodales libero, suscipit ullamcorper neque. Duis feugiat maximus faucibus. Praesent consectetur fermentum velit non eleifend. Cras nec lorem interdum, aliquam nulla sed, elementum arcu. Vivamus non maximus nisi. Duis quis lacinia sem, nec imperdiet sapien. Vivamus a ipsum ut nibh congue venenatis.
      </p>
    </div>
  </body>,
  "container": <div>
    <h2
      class="home-title"
    >
      Welcome to our page
    </h2>
    <p
      class="home-text"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sed vestibulum erat. Nulla sagittis massa at egestas facilisis. Quisque tincidunt interdum vulputate. Sed orci orci, viverra vel tincidunt vel, consectetur eget tortor. Integer pellentesque eros vel ex maximus, quis aliquet nisl tristique. Nulla lobortis sem a placerat maximus. Duis suscipit purus at nunc bibendum, et finibus quam tempor. Donec sed magna suscipit, bibendum nisi faucibus, dictum erat. Aenean varius eros vitae arcu tempus lobortis. Ut et nisl eget purus ornare tincidunt. Aliquam erat volutpat. Suspendisse vitae finibus odio. Suspendisse aliquam risus massa, vel vulputate mauris tempus a. In ullamcorper arcu eu suscipit facilisis. Suspendisse at elit et arcu finibus pulvinar. Fusce at ullamcorper libero. Nunc eget diam rutrum, lobortis sem vel, cursus lacus. Aliquam at elit nec est ultrices pharetra sit amet et dolor. Pellentesque mattis sodales libero, suscipit ullamcorper neque. Duis feugiat maximus faucibus. Praesent consectetur fermentum velit non eleifend. Cras nec lorem interdum, aliquam nulla sed, elementum arcu. Vivamus non maximus nisi. Duis quis lacinia sem, nec imperdiet sapien. Vivamus a ipsum ut nibh congue venenatis.
    </p>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
